api/src/services/AuthenticationService.ts

```diff
@@ -1,6 +1,6 @@
 import { Scope, UserStoredAttributes } from "../utils/types/attributeTypes";
 import * as argon2 from "argon2";
-import crypto from "crypto";
+import crypto, { UUID } from "node:crypto";
```

Comment:
Just a heads up on import mismatch, two different ways  to import Crypto. Different import is in user.model.ts

Reply:
Thankyou for spotting.